_posts/2020-02-27-integrating-with-a-third-party.md

```diff
@@ -0,0 +1,243 @@
+
+ - *Customer/Supplier models*, which establish an upstream-downstream relation. The Supplier team tailors its model specifically for the Customer model. For this strategy to work, the developers of both models have to collaborate.
+ - The *Conformist* pattern, which is very similar to the Customer/Supplier, except that the upstream is not available for change. Thus the downstream has to copy the language into its model thoughtlessly, hence conformism.
+ - *Anticorruption Layer* pattern (a.k.a. *ACL*), describing a Bounded Context, which is not willing to accept the language of the upstream and instead builds an intermediate model in the "no man’s land". The *ACL* translates the language of the upstream into the native language of the downstream Context.
+
+<p class="side-note">
+This list is not exhaustive. The ["Domain-Driven Design" book](https://dddcommunity.org/book/evans_2003/) offers a few more strategies, all worth considering. However, in this article, we are going to describe the listed three patterns, because they are the most commonly used.
+</p>
+
+## The Domain
+
+![Big picture domain]({{ site.baseurl }}/img/integrating-with-a-3d-party/domain.jpg)
+
+For the sake of an example, let’s consider airport management software. An airport is a complex system which relies on many people and much software working together. Let’s consider the system which helps the flight dispatchers make decisions on **Takeoffs and Landings**. The system integrates with the software responsible for **Security Checks**, **Airplane Supplies**, and **Weather**. All of these systems are independent of **Takeoffs and Landings** as well as of each other. Thus, each of them can be treated as a third party.
+
+## Customer/Supplier Contexts
+
+![Customer/Supplier Contexts domain]({{ site.baseurl }}/img/integrating-with-a-3d-party/customer-supplier.jpg){: .small-square-image}
+
+The **Takeoffs and Landings** system must know whether an *Aircraft* is ready for the *Flight*. This decision requires data on the supplies, which are provided for the *Aircraft*. To obtain this knowledge, the system integrates with the **Airplane Supplies** Context. Note the language difference between the *Aircraft* and an *Airplane*. Apparently, the two Contexts view the same entity of the real world from different perspectives.
+**Airplane Supplies** Context is an integral part of the airport software. Thus, by communicating with the developers responsible for **Airplane Supplies**, we are able to build a Customer/Supplier relationship between the **Airplane Supplies** Context and our system. The **Airplane Supplies** Context does not implement an Event-based messaging internally. However, it still acts as a Supplier. Specially for **Takeoffs and Landings**, the Supplier Context generates Events and publishes them to a shared channel. **Takeoffs and Landings**, the Customer, subscribes to those Events. 
```

Comment:
We need bigger text for the articles, and a serif font, please. Consider how Airbnb [looks like](https://medium.com/airbnb-engineering/building-an-effective-test-pipeline-in-a-service-oriented-world-6968c513c6bd).

Reply:
Please also have the same style for the background and main text borders as we have for the Documentation area.